pkg/testinfra/dsl/nfsInstance.go

```diff
@@ -79,6 +79,22 @@ func WithNfsInstanceScope(scopeName string) ObjAction {
 	}
 }
 
+func WithRemoteRef(name string) ObjAction {
```

Comment:
There is already `WithKcpIpRangeRemoteRef()` in `kcpIpRange.go`. Ideally we will have one func capable handling all different types. We can merge it this way, but let's have it in the back of our minds to fix it.

Reply:
This is not ip range remote ref. This is remote ref of SKR NFSVolume.

Reply:
That's exactly what Im talking about. You gave it generic name `WithRemoteRef()` implicating it's capable of setting RemoteRef to any cloud-control type, but in fact it's handling the `NfsInstance` only. We have an option to go with two func: WithKcpIpRangeRemoteRef and WithKcpNfsInstanceRemoteRef, each doing it's own type test, or go with one WithRemoteRef that's doing two type tests and handling both types, and in future when third type added it will be modified to handle that new type as well. We might even define an interface that has remote ref setting capabilities and then try to cast to that interface, which will reduce amount of type checks we have to do it that one generic all handling func